docs/setup-attach-cli.asciidoc

```diff
@@ -94,6 +94,12 @@ NOTE: The JVM arguments may contain sensitive information, such as passwords pro
 If provided, this program continuously runs and attaches to all running and starting JVMs which match the `--exclude` and `--include` filters.
 --
 
+*--no-fork* added[1.35.0]::
++
+--
+If provided, this program will not fork and attempt to attach directly as the current user.
```

Comment:
Do you think we should elaborate on reasons the end-user might need this ? Probably the most common example is when the attachment process is started by `root` user and the target JVM is using a non-root user. Because this is probably the only reason we have to fork, then maybe renaming this to `--no-fork-as-user` could make this a bit more explicit.

Reply:
Since it is a command line argument, I prefer to keep it short (and somewhat a familiar one).
But I will add this explanation to the description. Thanks for the advice

Reply:
Alternatively, you suggested the  `--switch-user` in https://github.com/elastic/apm-agent-java/issues/2713 also.

Reply:
Yes, but when implementing I thought it makes more sense that the config option name reflects the opt-in purpose, which is to avoid the forking command. 
It seems weird that the only reason to use a config option named `--switch-user` is to stop switching users...

Reply:
See extended description